runtime/java/src/common/Util.java

```diff
@@ -504,6 +506,16 @@ public static StringCatter determineSilverHomePath(Class<?> clazz) {
 		File home = new File(jarLocation).getParentFile().getParentFile();
 		return new StringCatter(home.getPath());
 	}
+
+	public static StringCatter getSilverVersion() {
+		String path = determineSilverHomePath(Util.class).toString()+"/jars/silver.compiler.composed.Default.jar";
+        try {
+            JarInputStream file = new JarInputStream(new FileInputStream(path));
+            return new StringCatter(file.getManifest().getMainAttributes().getValue(Name.IMPLEMENTATION_VERSION));
+        } catch (Throwable t) {
+            throw new RuntimeException("Failed to get Silver version.", t);
+        }
+    }
```

Comment:
```suggestion
		try {
			JarInputStream file = new JarInputStream(new FileInputStream(path));
			return new StringCatter(file.getManifest().getMainAttributes().getValue(Name.IMPLEMENTATION_VERSION));
		} catch (Throwable t) {
			throw new RuntimeException("Failed to get Silver version.", t);
		}
	}

```

spaces->tabs, to match rest of file